CollAction/Views/Manage/Index.cshtml

```diff
@@ -9,95 +9,152 @@
     ViewData["Title"] = "Manage your account";
 }
 
+<script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+
 <div id="manage-index">
+    
+    
     <div class="container">
+
         <h2>@ViewData["Title"].</h2>
         <p class="text-success">@Model.StatusMessage</p>
+        </br>
+
+        <div class="row">
+            <div class="container">
+
+                <div class="panel panel-default">
+                        <div class="panel-heading"><dt>Password</dt></div>
+                        <div class="panel-body">
+                            <div class="row">
+                                <div class="col-md-3">
+                                    <form asp-controller="Manage" asp-action="ChangePassword" method="get">
+                                        <input class="symetric_buttons" type="submit" value="Change Password" />
+                                    </form>
+                                    </div>
+                                <div class="col-md-9"></div>
+                            </div>
+                        </div>
+                </div>
+
+            </div>
+        </div>
+
+        <br/><br/>
+
+        <div class="row">
+            <div class="container">
+                <div class="panel panel-default">
+                        <div class="panel-heading"><dt>Newsletter Subscription:</dt></div>
+                        <div class="panel-body">
+                            <div class="row">
+                                <div class="col-md-12">
+                                        <form asp-controller="Manage" asp-action="NewsletterSubscription" method="post" class="form-horizontal">
+                                            <div asp-validation-summary="All" class="text-danger"></div>
+                                            <label asp-for="NewsletterSubscription" class="control-label"></label>
+                                            @Html.CheckBoxFor(m => m.NewsletterSubscription, new { onclick = "this.form.submit();"})
```

Comment:
This doesn't work anymore with CSP, see: https://github.com/CollActionteam/CollAction/pull/279